src/pages/home/home.ts

```diff
@@ -212,7 +221,10 @@ export class HomePage {
     // Used at the startup of Copay
     var argv = gui.App.argv;
     if (argv && argv[0]) {
-      this.handleOpenUrl(argv[0]);
+      // The timeout waits for the components to be initialized
+      setTimeout(() => {
```

Comment:
Isn't there a less brittle way to do this than setTimeout? At the very least it makes testing messy.

Reply:
In a future PR, which is almost ready, we will merge a fix that will allow us to eliminate the timeout, but it needs to be tested yet.